packages/@sfx/core/test/unit/common/index.test.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from '../../utils';
+import {
+  Core as CoreExport,
+  Plugin as PluginExport,
+  PluginMetadata as PluginMetadataExport,
+  PluginRegistry as PluginRegistryExport,
+} from '../../../src';
+import Core from '../../../src/core';
+import { Plugin, PluginMetadata, PluginRegistry } from '../../../src/plugin';
+import * as CoreUtils from '../../../src/utils/core';
+
+describe('Entry point', () => {
+  it('should export Core', () => {
+    expect(CoreExport).to.equal(Core);
+  });
+
+  it('should export the Plugin interface', () => {
+    // The following should cause a compiler error if the two interfaces are not equal.
+    const plugin_is_assignable_to_plugin_export: PluginExport = {} as Plugin;
+    const plugin_export_is_assignable_to_plugin: Plugin = {} as PluginExport;
+  });
+
+  it('should export the PluginMetadata interface', () => {
+    // The following should cause a compiler error if the two interfaces are not equal.
+    const plugin_metadata_is_assignable_to_plugin_metadata_export: PluginMetadataExport = {} as PluginMetadata;
+    const plugin_metadata_export_is_assignable_to_plugin_metadata: PluginMetadata = {} as PluginMetadataExport;
+  });
+
+  it('should export the PluginRegistry interface', () => {
+    // The following should cause a compiler error if the two interfaces are not equal.
+    const plugin_registry_is_assignable_to_plugin_registry_export: PluginRegistryExport = {} as PluginRegistry;
+    const plugin_registry_export_is_assignable_to_plugin_registry: PluginRegistry = {} as PluginRegistryExport;
```

Comment:
Any reason we're not using camelCase in this file?

Reply:
These variables aren't really being used for anything, but rather as descriptions for what's being tested. I thought it was more readable to use snake_case here, but if you want me to move the descriptions to a comment instead and call the variables something more generic, I can do that.